src/components/Menu/Menu.stories.tsx

```diff
@@ -3,6 +3,7 @@ import type { StoryObj, Meta } from '@storybook/react';
 import React from 'react';
 
 import { Menu } from './Menu';
+import type { MenuProps } from './Menu';
```

Comment:
good question. using the extraction was causing a collision with a named private interface in HeadlessUI code (something like `MenuPropsArgs` or so). By using the already-defined props, it avoids this reuse of the interface. 

I think it would only apply to HeadlessUI. so weird

Reply:
good question. using the extraction was causing a collision with a named private interface in HeadlessUI code (something like `MenuPropsArgs` or so). By using the already-defined props, it avoids this reuse of the interface. 

I think it would only apply to HeadlessUI. so weird